site2/docs/txn-use.md

```diff
@@ -34,6 +34,12 @@ This section provides an example of how to use the transaction API to send and r
       ```
       acknowledgmentAtBatchIndexLevelEnabled=true
       ```
+    If you want to enable deduplication for transaction, follow the steps below.
```

Comment:
Since it's a mandatory setting to enable transactions, I think it can be added to row28 following `systemTopicEnabled=true`.

The following descriptions can be added:
* `systemTopicEnabled=true` is a mandatory configuration. Otherwise, transactions cannot work because transaction buffer snapshots can’t be created.
*  `brokerDeduplicationEnabled=true` is a mandatory configuration to guanrantee exactly-once semantics.

BTW, does it make sense to add `standalone.conf` to row23?

Reply:
Great suggestion, I will write these configuration together and add descriptions for them